cms/apps/media/models.py

```diff
@@ -212,10 +318,103 @@ class Video(models.Model):
         null=True
     )
 
+    external_video = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        help_text='Please provide a youtube.com or vimeo.com URL',
```

Comment:
Drop the 'Please'. Also: Pretty much any provider is supported by this code (anything which supports oEmbed). Comments on that elsewhere.

Reply:
(Also, make it a full sentence with punctuation. If there's inconsistencies in the CMS on this count, I'd rather change the others.)